java/ingest/ingest-status-store/src/test/java/sleeper/ingest/status/store/job/StoreIngestJobValidatedIT.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2022-2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.ingest.status.store.job;
+
+import org.junit.jupiter.api.Test;
+
+import sleeper.ingest.job.IngestJob;
+import sleeper.ingest.job.status.ValidationData;
+import sleeper.ingest.status.store.testutils.DynamoDBIngestJobStatusStoreTestBase;
+
+import java.time.Instant;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static sleeper.ingest.job.status.IngestJobStatusTestData.invalidRun;
+import static sleeper.ingest.job.status.IngestJobStatusTestData.jobStatus;
+import static sleeper.ingest.job.status.IngestJobStatusTestData.startedIngestRunWithValidation;
+
+public class StoreIngestJobValidatedIT extends DynamoDBIngestJobStatusStoreTestBase {
+    @Test
+    void shouldReportIngestJobStartedWithValidation() {
+        // Given
+        IngestJob job = jobWithFiles("file1");
+        Instant validationTime = Instant.parse("2022-12-14T13:50:12.001Z");
+        Instant startedTime = Instant.parse("2022-12-14T13:51:12.001Z");
+
+        // When
+        store.jobValidated(DEFAULT_TASK_ID, job, validationTime, ValidationData.valid());
+        store.jobStarted(DEFAULT_TASK_ID, job, startedTime, true);
+
+        // Then
+        assertThat(getAllJobStatuses())
+                .usingRecursiveFieldByFieldElementComparator(IGNORE_UPDATE_TIMES)
+                .containsExactly(jobStatus(job,
+                        startedIngestRunWithValidation(job, DEFAULT_TASK_ID,
+                                validationTime, ValidationData.valid(), startedTime)));
```

Comment:
It's quite difficult to follow whether the flag is set or asserted correctly here on the job started record. Can this be made a bit more explicit?